src/test/java/org/folio/circulation/rules/Text2DroolsTest.java

```diff
@@ -432,18 +461,41 @@ public void shelvingLocation() {
       createLocation(FIRST_INSTITUTION_ID, FIRST_LIBRARY_ID, FIRST_CAMPUS_ID)),   is("policy-e"));
   }
 
+  @Test
+  public void overdueFinePolicy() {
+    Drools drools = new Drools(Text2Drools.convert(String.join("\n",
+      "priority: last-line",
+      "fallback-policy: l no-loan r no-hold n basic-notice o fallback",
+      "s new: l policy-a r no-hold n basic-notice o overdue-a",
+      "m book: l policy-b r no-hold n basic-notice o overdue-b",
+      "a " + FIRST_INSTITUTION_ID + ": l policy-c r no-hold n basic-notice o overdue-c",
+      "b new: l policy-d r no-hold n basic-notice o overdue-d",
+      "c "+ FIRST_LIBRARY_ID + ": l policy-e r no-hold n basic-notice o overdue-e",
+      "b "+ FIRST_CAMPUS_ID + ": l policy-e r no-hold n basic-notice o overdue-e")));
```

Comment:
Can you rename the second overdue-e to overdue-f and add a test so that both of them (overdue-e and overdue-f) get tested?

Reply:
Done.